src/sidebar/components/ShareDialog/test/FileInput-test.js

```diff
@@ -0,0 +1,107 @@
+import { mount } from 'enzyme';
+
+import FileInput from '../FileInput';
+
+describe('FileInput', () => {
+  let wrappers;
+  let container;
+  let fakeOnFileSelected;
+
+  const findNativeFileInputInDOM = () =>
+    document.querySelector('input[data-testid="file-input"]');
```

Comment:
Was there a particular reason for using `document.querySelector` rather than using `wrapper.find`, eg. `wrapper.find('input[type="file"]').getDOMNode()` to get the DOM node?

Reply:
I was going to say yes, that the wrapper is not available in all contexts, but I just realized that's actually not true 😅 

I'll use that. We don't need two functions doing almost the same.